edb/schema/defines.py

```diff
@@ -19,9 +19,15 @@
 
 from __future__ import annotations
 
-# Maximum length of column name in Postgres. This limits the database
-# name length and affects column name mangling.
-MAX_NAME_LENGTH = 63
+# Maximum length of Postgres tenant ID.
+MAX_TENANT_ID_LENGTH = 1 + 10
```

Comment:
Do you maybe want to reserve yet another 11 characters for future use, reducing the max length to 40? We won't be able to do this ever again.

Reply:
I don't think we need to reserve more.